signup.html

```diff
@@ -70,42 +71,48 @@ <h5 class="offcanvas-title" id="offcanvasNavbarLabel">SummerTrails Menu</h5>
         <h2>Signup Form</h2>
         <form action="#" method="POST">
             <div class="form-group">
-              <label for="emailAddress">Email address</label>
-              <input type="email" class="form-control" name="emailAddress" id="emailAddress" aria-label="Enter Email" placeholder="Enter email">
+                <label for="emailAddress">Email address</label>
+                <input type="email" class="form-control" name="emailAddress" id="emailAddress" aria-label="Enter Email"
```

Comment:
[nitpick] there seem to be some format changes in this document. Other than that it should be good to pull